docker/server.js

```diff
@@ -1,14 +1,12 @@
 const express = require('express')
 const app = express()
 
-const NodeHL7Complete = require('node-hl7-complete') // Obviously point to internals
+const NodeHL7Complete = require('node-hl7-complete')
 const nodeHL7Instance = new NodeHL7Complete()
 
-// TODO: This is crazy! So many API changes. I'm using 'body' for
-//       everything (for now). May be best to look for a totally 
-//       new middleware for this...
-const bodyParser = require('body-parser')
-app.use(bodyParser.text());
+// Latest Express middleware approach
+app.use(express.json())
+app.use(express.text())
```

Comment:
only concern of mine in this PR is, why do we have 2 middlewares for body parsing, wouldn't `express.json()` do?